plugins/routed-eni/driver/driver.go

```diff
@@ -233,30 +233,22 @@ func setupNS(hostVethName string, contVethName string, netnsPath string, addr *n
 			}
 			log.Infof("Added rule priority %d from %s table %d", fromContainerRulePriority, addr.String(), table)
 		} else {
-			// add rule: 1536: list of from <podIP> to <vpcCIDR> use table <table>
-			for _, cidr := range vpcCIDRs {
```

Comment:
The rule needs to be set for all CIDR ranges in the VPC I can't see how this is supposed to work.